core/src/main/java/org/frankframework/management/bus/endpoints/ServiceListener.java

```diff
@@ -52,6 +55,7 @@ public Message<String> getServiceListeners(Message<?> message) {
 
 	@TopicSelector(BusTopic.SERVICE_LISTENER)
 	@ActionSelector(BusAction.UPLOAD)
+	@RolesAllowed({"IbisDataAdmin", "IbisAdmin", "IbisTester"})
```

Comment:
Deze moet wel, zie https://github.com/frankframework/frankframework/blob/7.5-release/webapp/src/main/webapp/WEB-INF/web.xml#L360
```suggestion
	@RolesAllowed("IbisTester")
```